src/main/java/com/epam/wl/services/UserOrderService.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.wl.services;
+
+import com.epam.wl.dao.BookDAO;
+import com.epam.wl.dao.UserOrderDAO;
+import com.epam.wl.db.JdbcConnector;
+import com.epam.wl.entities.UserOrder;
+import com.epam.wl.enums.UserOrderStatus;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabase;
+
+import javax.sql.DataSource;
+import java.sql.SQLException;
+import java.util.*;
+
+public class UserOrderService {
+
+    private static UserOrderService instance;
+//    private static DataSource dataSource = MockDBHelper.getEmbeddedDatabase();
```

Comment:
commented code shouldn't exist - you can always use git to find something you changed